### PR TITLE
Feature play with friends

### PR DESCRIPTION
**Pull request by nasimafeef**

### CODE REVIEW

File: src/Application.js [hunk 1]
```diff
@@ -186,14 +189,63 @@ exports = Class(GC.Application, function () {
...
+
+    popup.build({
+      superview: game,
+      device: {
+        width: size.width,
+        height: size.height
+      },
+      popup: {
+        width: size.width * 85 / 100,
+        height: size.height * 40/100,
```
Comment (rampr) [inline src/Application.js]: why are there additional calculations for width and height ?

File: src/Popups/popup_common.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+      superview: this.container,
+      layout: 'linear',
+      direction: 'vertical',
+      width: width,
+      justifyContent: 'center',
+      layoutHeight: 'wrapContent',
+      margin: 30
+    });
+    
+    this.panel_contents = [];
```
Comment (vivekb91) [inline src/Popups/popup_common.js]: can it be a local var?

File: src/Popups/popup_common.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+import util.underscore as _;
+import ui.TextView as TextView;
+import ui.ImageScaleView as ImageScaleView;
+
+import src.Utils as Utils;
+/* jshint ignore:end */
+
+exports = Class(View, function(supr) {
+  this.init = function() {
+  	supr(this, 'init');
```
Comment (vivekb91) [inline src/Popups/popup_common.js]: over indentation

File: src/Application.js [hunk 4]
```diff
@@ -186,14 +189,80 @@ exports = Class(GC.Application, function () {
...
+                }
+              },
+            ]
+          };
+        });
+        connectedPlayers = _.shuffle(connectedPlayers)
+          .slice(0, 10);
+
+        return cb({
+            title: 'Play with friends',
```
Comment (vivekb91) [inline src/Application.js]: extra indentation

File: src/Popups/popup_common.js [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+  var width;
+
+  this.build = function(opts) {
+    var parent_view = opts.superview,
+      popup_opts = opts.popup,
+      close_button, base_height, extra;
+
+    width = popup_opts.width;
+    this.overlay = new ScrollView({
```
Comment (rampr) [inline src/Popups/popup_common.js]: why should the overlay be a scrollview ?

**Pull request merged**